src/DataFactory/DataFactoryV2/help/Update-AzDataFactoryV2.md

```diff
@@ -15,18 +15,27 @@ Updates the properties of a data factory.
 ### ByFactoryName (Default)
 ```
 Update-AzDataFactoryV2 [-ResourceGroupName] <String> [-Name] <String> [-Tag <Hashtable>]
+ [-IdentityType] <String>
```

Comment:
IdentityType is optional why its syntax says it is required? Have you updated this markdown by https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/help-generation.md#updating-a-single-markdown-file?

Reply:
@BethanyZhou my team was always modifying that file manually. I have followed the steps for both markdown files and updated the PR